cluster.go

```diff
@@ -125,6 +132,33 @@ func (cfg *ClusterConfig) CreateSession() (*Session, error) {
 	return NewSession(*cfg)
 }
 
+// translateHostPort is a helper method that will use the given AddressTranslator
+// if defined, to translate the given host:port (addr) into a new host:port string.
+// If no AddressTranslator or if an error occurs, the given host:port will be returned.
+func (cfg *ClusterConfig) translateHostPort(hostPort string) (string) {
+	if cfg.AddressTranslator == nil {
+		return hostPort
+	}
+	host, port, err := parseHostPort(hostPort)
+	if err != nil {
+		return hostPort
+	}
+	newHost, newPort := cfg.AddressTranslator.Translate(host, port)
```

Comment:
I get the defensiveness, but it feels unnecessary to me. I've update the documentation for `Translate` to state that the provided host:port should be returned if no translation is possible.